src/conversation-store.ts

```diff
@@ -69,7 +69,8 @@ export function conversationContext<ConversationState = any>(
         .catch((error) => {
           logger.debug(`Conversation context not loaded: ${error.message}`);
         })
-        .then(next);
+        .then(next)
+        .catch(error => logger.debug(error.message));
```

Comment:
there wasn't a final catch here, which was resulting in the tslint error.

```
ERROR: /Users/nick.bartlettibm.com/dev/bolt/src/conversation-store.ts:64:7 - Promises must be handled appropriately
```

Reply:
seems silly, but also seems like a harmless way to satisfy tslint 😄